<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QDialog" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>657</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Javanese Text</family>
    <pointsize>11</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>設定</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(68, 68, 68);
color: rgb(255, 255, 255);
font: 11pt &quot;Javanese Text&quot;;</string>
  </property>
  <widget class="QRadioButton" name="radioButton_type_1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>時間指定注文</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_type_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>40</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>価格監視注文</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_type_3">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>40</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>在庫監視注文</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_save">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>600</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>600</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>閉じる</string>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendar">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>600</y>
     <width>309</width>
     <height>251</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(204, 204, 204);
font: 8pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>90</y>
     <width>451</width>
     <height>491</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <widget class="QLineEdit" name="lineEdit_url">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>5</y>
         <width>311</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>8</x>
         <y>5</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>製品のURL</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_dateTime" native="true">
      <widget class="QTimeEdit" name="timeEdit">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>7</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(203, 203, 203);
color: rgb(0, 0, 0);</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="displayFormat">
        <string>HH:mm</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_dateTime">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>7</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>購入時間</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="dateEdit">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>7</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(203, 203, 203);
color: rgb(0, 0, 0);</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="accelerated">
        <bool>true</bool>
       </property>
       <property name="displayFormat">
        <string>yyyy-MM-dd</string>
       </property>
      </widget>
      <widget class="QToolButton" name="calendar_button">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>7</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_price" native="true">
      <widget class="QLineEdit" name="lineEdit_price">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>5</y>
         <width>311</width>
         <height>31</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>5</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>購入価格</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_5" native="true">
      <widget class="QSpinBox" name="spinBox_count">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>4</y>
         <width>311</width>
         <height>31</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>4</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>購入個数</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_6" native="true">
      <widget class="QSpinBox" name="spinBox_loops">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>5</y>
         <width>311</width>
         <height>31</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>5</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>購入回数</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_7" native="true">
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>11</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>サイズ</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_size">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>10</y>
         <width>311</width>
         <height>31</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <widget class="QLineEdit" name="lineEdit_color">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>5</y>
         <width>311</width>
         <height>31</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>5</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>カラー</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>radioButton_type_1</zorder>
  <zorder>radioButton_type_2</zorder>
  <zorder>radioButton_type_3</zorder>
  <zorder>pushButton_save</zorder>
  <zorder>pushButton_cancel</zorder>
  <zorder>calendar</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>calendar</sender>
   <signal>clicked(QDate)</signal>
   <receiver>Form</receiver>
   <slot>calendar_click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>275</x>
     <y>305</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>calendar_button</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>calendar_show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>285</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_save</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>620</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_cancel</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>cancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>620</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_type_1</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>type_1_click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>95</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_type_2</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>type_2_click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_type_3</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>type_3_click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>385</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>type_1_click()</slot>
  <slot>type_2_click()</slot>
  <slot>type_3_click()</slot>
  <slot>calendar_click()</slot>
  <slot>save()</slot>
  <slot>cancel()</slot>
  <slot>calendar_show()</slot>
 </slots>
</ui>
